src/ai/backend/manager/config/local.py

```diff
@@ -0,0 +1,935 @@
+        This database must exist and be accessible by the configured user.
+        Length must be between 2 and 64 characters due to database naming constraints.
+        """,
+        examples=["backend"],
+    )
+    user: str = Field(
+        description="""
+        Username for authenticating with the database.
+        This user must have sufficient privileges for all database operations.
+        """,
+        examples=["postgres"],
+    )
+    password: str = Field(
+        description="""
+        Password for authenticating with the database.
+        Can be a direct password string or an environment variable name.
+        For security, using environment variables is recommended in production.
+        """,
+        examples=["develove", "DB_PASSWORD"],
+    )
+    pool_size: int = Field(
+        default=8,
+        ge=1,
+        description="""
+        Size of the database connection pool.
+        Determines how many concurrent database connections to maintain.
+        Should be tuned based on expected load and database server capacity.
+        """,
+        examples=[1, 8],
+    )
+    pool_recycle: float = Field(
+        default=-1,
+        ge=-1,
+        description="""
+        Maximum lifetime of a connection in seconds before it's recycled.
+        Set to -1 to disable connection recycling.
+        Useful to handle cases where database connections are closed by the server after inactivity.
+        """,
+        examples=[-1, 50],
+    )
+    pool_pre_ping: bool = Field(
+        default=False,
+        description="""
+        Whether to test connections with a lightweight ping before using them.
+        Helps detect stale connections before they cause application errors.
+        Adds a small overhead but improves reliability.
+        """,
+        examples=[True],
+    )
+    max_overflow: int = Field(
+        default=64,
+        ge=-1,
+        description="""
+        Maximum number of additional connections to create beyond the pool_size.
+        Set to -1 for unlimited overflow connections.
+        These connections are created temporarily when pool_size is insufficient.
+        """,
+        examples=[-1, 64],
+    )
+    lock_conn_timeout: float = Field(
+        default=0,
+        ge=0,
+        description="""
+        Timeout in seconds for acquiring a connection from the pool.
+        0 means wait indefinitely.
+        If connections cannot be acquired within this time, an exception is raised.
+        """,
+        examples=[0, 30],
+    )
+
+
+_max_num_proc = os.cpu_count() or 8
+_file_perm = (Path(__file__).parent / "server.py").stat()
+
+
+class EventLoopType(enum.StrEnum):
+    asyncio = "asyncio"
+    uvloop = "uvloop"
+
+
+class DistributedLockType(enum.StrEnum):
+    filelock = "filelock"
+    pg_advisory = "pg_advisory"
+    redlock = "redlock"
+    etcd = "etcd"
+    etcetra = "etcetra"
+
+
+class EtcdConfig(BaseModel):
+    namespace: str = Field(
+        default="backend",
```

Comment:
It seems that there is no default value set for this configuration in the existing code.
Was it intentional to set the default value on the backend? @HyeockJinKim